MapCanvas.py

```diff
@@ -125,12 +125,12 @@ def __init__(self, master, stage_data=None):
         self.canvas.bind('<B1-Motion>', self.drag_block)
         self.canvas.bind('<ButtonPress-3>', self._show_popup)
 
-        self.canvas.bind('<Left>', self.scroll_left)
-        self.canvas.bind('<Right>', self.scroll_right)
-        self.canvas.bind('<Up>', self.scroll_up)
-        self.canvas.bind('<Down>', self.scroll_down)
+        self.canvas.bind('<Left>', lambda event: self.canvas.xview_moveto(self.canvas.xview()[0] - 1 / self.width))
```

Comment:
a) why?
b) these lines are too long (>80 characters)

Reply:
I wasn't entirely sold on having those long ones either, but I thought it was worth a shot. Are the short ones okay, at least? I'll revert the long ones. The majority of the lambdas was just so we don't have functions for the sole purpose of calling other ones.

Reply:
I guess the way I had it it is a bit clearer, and if you wanted to add some checks or something in the scroll functions you could do it more easily there.
It isn't too uncommon to see stuff like this I think...

Reply:
Hmm, alright, I'll revert that back tomorrow then. Let me know if there's anything else to do. Don't worry about merge compatibility either, I'll take care of that before I set it to [MRG]